cpp_exercises/day2/date/date.h

```diff
@@ -63,6 +64,8 @@ class Date
     static bool isYearValid(int year);
     static bool isMonthValid(int month);
     static bool isDateValid(int day, int month, int year);
+    static bool areStringValidInteger(const std::string& string);
```

Comment:
Questo commento te lo lascio per quanto riguarda il naming.
`areStringValidInteger` non mi suona molto corretto come nome, in primis perchè usi il verbo plurale (are) con un sostantivo singolare (string anzichè strings).
Una cosa che faccio solitamente è provare i nomi in Google Translate per vedere come me li traduce dall'inglese all'italiano come check aggiuntivo, in questo caso secondo Google il significato sarebbe `sono numeri interi validi per le stringhe`, traduzione che comunque si è un po' aggiustato interpretando il testo originale come `are valid strings for integers`.
In questo caso io la funzione l'avrei semplicemente chiamata `isValidInteger(...)` omettendo la parola `string` nel nome, perchè tanto sarebbe stato ovvio dal parametro.